dev/fattest.simplicity/src/componenttest/topology/utils/ExternalTestService.java

```diff
@@ -561,9 +561,16 @@ public void writePropertyAsFile(String keyName, File file) throws IOException {
             throw new IllegalStateException("Key not found in service properties: " + keyName);
         }
 
+        String decryptedProp = null;
+        try {
+            decryptedProp = prop.getStringValue();
```

Comment:
I think this will break some mongodb tests, as they have some properties that contain non-UTF-8 characters

Perhaps we can do some switch based on if its encrypted

Reply:
good point. I've undone this change and instead done `externalService.getProperties().getProperty(propName)` (which does already decrypt) and then I write that string to a file using `java.io.Files`